components/gcp-click-to-deploy/src/DeployForm.tsx

```diff
@@ -144,7 +144,7 @@ export default class DeployForm extends React.Component<any, DeployFormState> {
   public render() {
     const zoneList = ['us-central1-a', 'us-central1-c', 'us-east1-c', 'us-east1-d', 'us-west1-b',
       'europe-west1-b', 'europe-west1-d', 'asia-east1-a', 'asia-east1-b'];
-    const versionList = ['v0.3.5', 'v0.4.0'];
```

Comment:
why is 0.4.0 removed?

Reply:
We'll overwrite versions through env var. https://github.com/kubeflow/kubeflow/blob/master/components/gcp-click-to-deploy/src/DeployForm.tsx#L191
List here is for test only, and v0.4.0's registry file is not up to date